src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -26,7 +26,7 @@ export class GroupInviteUsersComponent implements OnInit, OnDestroy {
 
   readonly inputName = 'logins'
   state: 'empty'|'too_many'|'loading'|'ready' = 'empty';
-  inviteForm = this.formBuilder.group({ invitations: '' });
+  inviteForm = this.formBuilder.group({ logins: '' });
```

Comment:
That shows that repeating the constant "logins" here was not a good idea :-(

Reply:
I removed the constant